diff --git a/src/cgnstools/cgnsplot/CMakeLists.txt b/src/cgnstools/cgnsplot/CMakeLists.txt
index 80293a0..8885d1e 100644
--- a/src/cgnstools/cgnsplot/CMakeLists.txt
+++ b/src/cgnstools/cgnsplot/CMakeLists.txt
@@ -50,7 +50,7 @@ else ()
   list(APPEND PLOT_LK_LIBS ${X11_X11_LIB} ${X11_Xmu_LIB} ${X11_SM_LIB} m)
 endif ()
 
-target_link_libraries(plotwish PRIVATE tkogl ${PLOT_LK_LIBS} OpenGL::GL)
+target_link_libraries(plotwish PRIVATE tkogl ${PLOT_LK_LIBS})
 
 #target_include_directories(plotwish PRIVATE ${TCL_INCLUDE_PATH} ${TK_INCLUDE_PATH} ${OPENGL_INCLUDE_DIR})
 #if (X11_FOUND)
diff --git a/src/cgnstools/tkogl/CMakeLists.txt b/src/cgnstools/tkogl/CMakeLists.txt
index 730aa8b..27901ff 100644
--- a/src/cgnstools/tkogl/CMakeLists.txt
+++ b/src/cgnstools/tkogl/CMakeLists.txt
@@ -23,7 +23,7 @@ if (X11_FOUND)
   list(APPEND TKOGL_LIBS ${X11_GL_LIB} ${X11_GLU_LIB})
 endif()
 
-target_link_libraries(tkogl PUBLIC ${TKOGL_LIBS} OpenGL::GL OpenGL::GLU)
+target_link_libraries(tkogl PUBLIC ${TKOGL_LIBS})
 
 target_include_directories(tkogl PUBLIC ${TCL_INCLUDE_PATH} ${TK_INCLUDE_PATH} ${OPENGL_INCLUDE_DIR}) 
 target_include_directories(tkogl PUBLIC ../common)
